src/solution_summary.jl

```diff
@@ -118,6 +120,7 @@ function _show_candidate_solution_summary(io::IO, summary::_SolutionSummary)
         "  Objective bound      : ",
         summary.objective_bound,
     )
+    _print_if_not_missing(io, "  Objective gap        : ", summary.relative_gap)
```

Comment:
It should be clear that its relative and not absolute

Reply:
Yeah I renamed to "Relative gap"